lib/ansible/modules/cloud/amazon/ec2_key.py

```diff
@@ -95,6 +95,25 @@
     state: absent
 '''
 
+RETURN = '''
+key.fingerprint:
```

Comment:
@bcoca It appears that the subkeys of complex types currently aren't rendered by the docs (e.g.,  [docker_service][1], [os_port_facts][2]). I'm loathe to make a change that means this info won't be visible in the docs. Should I submit a separate ticket for the issue of subkeys not being rendered? 

[1]: https://docs.ansible.com/ansible/docker_service_module.html#return-values
[2]: https://docs.ansible.com/ansible/os_port_facts_module.html#return-values